.github/workflows/containerized-ci.yml

```diff
@@ -1,84 +1,123 @@
+    - documentation
+
+defaults:
+  run:
+    shell: bash
 
 jobs:
   Containerized-CI:
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
+        fortran-compiler: [ifort, nvfortran]
         rte-kernels: [default, openacc]
-        container: ["earthsystemradiation/rte-rrtmgp-ci:ifort","earthsystemradiation/rte-rrtmgp-ci:nvfortran"]
+        include:
+        - fortran-compiler: ifort
+          fcflags: "-m64 -g  -traceback -heap-arrays -assume realloc_lhs -extend-source 132 -check bounds,uninit,pointers,stack -stand f08"
+          image: "earthsystemradiation/rte-rrtmgp-ci:ifort"
+        - fortran-compiler: nvfortran
+          fcflags: "-Mallocatable=03 -Mstandard -Mbounds -Mchkptr -Kieee -Mchkstk"
+          image: "earthsystemradiation/rte-rrtmgp-ci:nvfortran"
+        - netcdf-c: /home/runner/netcdf-c
+        - netcdf-fortran: /home/runner/netcdf-fortran
     container:
-      image: ${{ matrix.container }}
+      image: ${{ matrix.image }}
     env:
-      NCHOME: /home/runner/netcdf-c
-      NFHOME: /home/runner/netcdf-fortran
-      RFMIP_DIR: /home/runner/rfmip-files
+      # Core variables:
+      FC: ${{ matrix.fortran-compiler }}
+      FCFLAGS: ${{ matrix.fcflags }}
+      # Make variables:
+      NCHOME: ${{ matrix.netcdf-c }}
+      NFHOME: ${{ matrix.netcdf-fortran }}
+      RRTMGP_ROOT: ${{ github.workspace }}
+      RTE_KERNELS: ${{ matrix.rte-kernels }}
+      RUN_CMD:
+      # Auxiliary variables:
+      RFMIP_CACHEDIR: /home/runner/rfmip-files
     steps:
-    ############################################################################
-    # Checks out repository under $GITHUB_WORKSPACE
-    - name: Check out code
-      uses: actions/checkout@v3
-    - name: Environmental variables
-      # This might be able to be set in the ENV section above
-      run: echo "RRTMGP_ROOT=${GITHUB_WORKSPACE}"        >> $GITHUB_ENV
-    - name: Environmental variables - ifort
-      if: contains(matrix.container, 'ifort')
-      run: echo "FCFLAGS=-m64 -g  -traceback -heap-arrays -assume realloc_lhs -extend-source 132 -check bounds,uninit,pointers,stack -stand f08" >> $GITHUB_ENV
-    - name: Environmental variables - nvfortran
-      if: contains(matrix.container, 'nvfortran')
-      run: echo "FCFLAGS=-Mallocatable=03 -Mstandard -Mbounds -Mchkptr -Kieee -Mchkstk" >> $GITHUB_ENV
-
-    - name: Make library, examples, tests
-      shell: bash
-      env:
-        RTE_KERNELS: ${{ matrix.rte-kernels }}
+    #
+    # Checks-out repository under $GITHUB_WORKSPACE
+    #
+    - uses: actions/checkout@v3
+    #
+    # Install Zstd
+    #
+    - name: Install Zstd
       run: |
-        source /opt/intel/oneapi/setvars.sh || true
-        cd ${RRTMGP_ROOT}
-        ${FC} --version
-        make libs
-        make -C build separate-libs
-    ############################################################################
+        # We skip 'apt-get update' to keep this step lightweight. However, that
```

Comment:
@skosukhin The containers have been updated to install `zstd`. Can this step be removed?